src/scripts/main.js

```diff
@@ -1,3 +1,76 @@
 'use strict';
 
-// write code here
+const appendRowBtn = document.querySelector('.append-row');
+const removedRowBtn = document.querySelector('.remove-row');
+const appendColumnBtn = document.querySelector('.append-column');
+const removeColumnBtn = document.querySelector('.remove-column');
+const sizeOfTable = {
+  maxCountRows: 10,
+  maxCountColumns: 10,
+  minCountRows: 2,
+  minCountColumns: 2,
+};
+
+appendRowBtn.addEventListener('click', function(e) {
```

Comment:
```suggestion
appendRowBtn.addEventListener('click', (e) => {
```
It is better to use arrow functions here

Reply:
if i use arrow functions, 'this' doesn't work
i use 'this' in add an attribute  'disabled'